tests/Makefile

```diff
@@ -13,22 +13,22 @@ PROGS = test1 test2 test3 test4 test5 test6 test7 test8 test9   \
         test74 test75 test76 test77 test78 test79 test80 test81 \
         test82 test83 test84 test85 test86 test87 test88 test89 \
         test90 test91 test92 test93 test94 test95
-CFLAGS += -I$(HASHDIR)
-#CFLAGS += -DHASH_BLOOM=16
-#CFLAGS += -O2
-CFLAGS += -g
-#CFLAGS += -Wstrict-aliasing=2
-CFLAGS += -Wall
-#CFLAGS += -Wextra
-#CFLAGS += -std=c89
-CFLAGS += ${EXTRA_CFLAGS}
+UTH_CFLAGS += -I$(HASHDIR)
+#UTH_CFLAGS += -DHASH_BLOOM=16
+#UTH_CFLAGS += -O2
+UTH_CFLAGS += -g
+#UTH_CFLAGS += -Wstrict-aliasing=2
+UTH_CFLAGS += -Wall
+#UTH_CFLAGS += -Wextra
+#UTH_CFLAGS += -std=c89
+UTH_CFLAGS += ${EXTRA_CFLAGS}
```

Comment:
Besides my high-level comments in the "Conversation" tab: should this now use `${CFLAGS}` instead of `${EXTRA_CFLAGS}`? I believe the purpose of `EXTRA_CFLAGS` was so that the user can pass their own extra stuff, but if we're going to respect their `CFLAGS` then they don't need this extra mechanism.